### PR TITLE
Change background color of the calendar when the dates are published

### PR DESCRIPTION
**Pull request by jeffreybos-timeseries**

### CODE REVIEW

File: packages/customWidgets/calendar-web/src/components/CalendarContainer.ts [hunk 1]
```diff
@@ -151,12 +153,27 @@ export default class CalendarContainer extends Component<Container.CalendarConta
         return new Date();
     };
 
+    private getDisabledTill = async (mxObject: MxObject): Promise<Date> => {
+        if (mxObject && this.props.disabledTill) {
+            let disabledTill;
+            try {
+                disabledTill = await this.extractAttributeValue<number>(mxObject, this.props.disabledTill);
+            } catch (error) {
+                window.mx.ui.error(`Unable to fetch start date attribute value: ${error.message}`);
```
Comment (ghost) [inline packages/customWidgets/calendar-web/src/components/CalendarContainer.ts]: "start date" --> "disabledTill date"

File: packages/customWidgets/calendar-web/src/components/Calendar.ts [hunk 2]
```diff
@@ -199,6 +201,12 @@ class Calendar extends Component<CalendarProps, State> {
 
     private eventColor = (events: CalendarEvent) => ({ style: { backgroundColor: events.color } });
 
+    private dayGetter = (date: Date) => {
+        return this.props.disabledTill && this.props.disabledTill !== new Date("") && this.props.disabledTill > date
+            ? { className: "rbc-disabled abc", style: {} }
```
Comment (ghost) [inline packages/customWidgets/calendar-web/src/components/Calendar.ts]: class abc?

File: packages/customWidgets/calendar-web/src/components/CalendarContainer.ts (hunk 1 above)
Comment (jeffreybos-timeseries) [reply packages/customWidgets/calendar-web/src/components/CalendarContainer.ts]: Misread, i changed the attribute to disabledTillDate

File: packages/customWidgets/calendar-web/src/components/Calendar.ts (hunk 2 above)
Comment (jeffreybos-timeseries) [reply packages/customWidgets/calendar-web/src/components/Calendar.ts]: It was a copy/paste from Freeks solution. I removed the 'abc' class

**Pull request merged**